README.md

```diff
@@ -13,6 +13,29 @@ Pigeonhole requires the following:
 
 After these have been installed, copy the config.toml.example file to config.toml, and update it with your details.
 
+Pigeonhole operates as a ro user with the exception of the categorisation. If you're using the categorisation feature you'll have to add a rw user as well.
```

Comment:
Given that this is public, I'd:
`s/ro user/read-only user/ s/rw user/read-write user/`